ncn_civi_zoom.php

```diff
@@ -174,6 +175,27 @@ function ncn_civi_zoom_civicrm_preProcess($formName, &$form) {
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_navigationMenu
  *
+ */
+function ncn_civi_zoom_civicrm_navigationMenu(&$menu) {
+  $parentId             = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Navigation', 'Events', 'id', 'name');
+  $maxId                = max(array_keys($menu));
+  $zoomSettingMaxId     = $maxId+1;
+
+  $menu[$parentId]['child'][$zoomSettingMaxId] = array(
+        'attributes' => array(
+          'label'     => ts('Zoom Settings'),
+          'name'      => 'Zoom_Settings',
+          'url'       => 'civicrm/Zoom/settings?reset=1',
```

Comment:
Use CRM_Utils_System::url() function to generate url. Because hard url format might not be work as expected in wordpress sites. so please always use civi functions to generate url rather than hard coded.